src/Client.js

```diff
@@ -155,9 +155,21 @@ var values = require('./values')
  * @param {?fetch} options.fetch
  *   a fetch compatible [API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) for making a request
  * @param {?number} options.queryTimeout
- *   Sets the maximum amount of time (in milliseconds) for query execution on the server,
+ *   Sets the maximum amount of time (in milliseconds) for query execution on the server
+ * @param {?number} options.http2SessionIdleTime
+ *   Sets the maximum amount of time (in milliseconds) an HTTP2 session may live
+ *   when there's no activity. Only applicable for NodeJS environment (when http2 module is used), 500 by default,
+ *   cab be configured via FAUNADB_HTTP2_SESSION_IDLE_TIME environment variable.
```

Comment:
typo: `cab` should be `can`
```suggestion
 *   can be configured via FAUNADB_HTTP2_SESSION_IDLE_TIME environment variable.
```